app/src/main/java/ar/com/wolox/android/example/ui/news/NewsFragment.kt

```diff
@@ -43,6 +44,9 @@ class NewsFragment @Inject constructor() : WolmoFragment<NewsPresenter>(), INews
     }
 
     override fun setNews(newsList: List<News>) {
+        vNoNewsText.apply {
```

Comment:
If you're changing only one attribute of the view, it's not necesssary to use `apply{}`